src/frontegg-auth/src/error.rs

```diff
@@ -22,6 +22,8 @@ pub enum Error {
     ReqwestError(Arc<reqwest::Error>),
     #[error("middleware programming error: {0}")]
     MiddlewareError(Arc<anyhow::Error>),
+    #[error("request to Frontegg failed: {0:?}")]
+    FronteggError(Arc<(reqwest::Error, String)>),
```

Comment:
I don't see the second part here, the body, ever being read. Is it needed for something?